src/StatefulComponent34.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from "react";
+
+export default function StatefulComponent34() {
+  const [count, setCount] = useState(0);
+  function increment() {
+    setCount(count + 1);
```

Comment:
setCount(count + 1)
Šitą eilutę galima kiek kitaip geriau parašyti. Siūlau paieškoti, kaip updatinti react state jeigu ji priklauso nuo prieš tai buvusios vertės.